src/strands/multiagent/graph.py

```diff
@@ -570,15 +700,34 @@ async def _execute_node(self, node: GraphNode, invocation_state: dict[str, Any])
                     )
 
                 elif isinstance(node.executor, Agent):
+                    # For agents, stream their events and collect result
+                    agent_response = None
                     if self.node_timeout is not None:
```

Comment:
Same comments as above - these if blocks have the same content, so let's find a way to unify them